src/tests/unit/tests/reports/package/__snapshots__/integration.test.ts.snap

```diff
@@ -9136,12 +9136,23 @@ exports[`report package integration with no issues 1`] = `
                     <span
                       class="guidance-links"
                     >
```

Comment:
This change can be compared with the source here before `link-name` was disabled: https://raw.githubusercontent.com/microsoft/accessibility-insights-web/f4ae57d42d1be911b055d327868a1975a8e615e0/src/tests/unit/tests/reports/package/__snapshots__/integration.test.ts.snap (ctrl-f for `link-name`)